app/views/addSupportingDocuments.scala.html

```diff
@@ -0,0 +1,60 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import controllers.routes._
+@import models.Mode
+@import uk.gov.hmrc.play.views.html._
+@import views.ViewUtils._
+
+@(appConfig: FrontendAppConfig, form: Form[_], goodsName: String, mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = s"${errorPrefix(form)} ${messages("addSupportingDocuments.title")}",
+    appConfig = appConfig,
+    bodyClasses = None) {
+
+    @helpers.form(action = AddSupportingDocumentsController.onSubmit(mode)) {
```

Comment:
Is this missing `@CSRF.formField`?

Reply:
I think helpers.form(...) calls CSRF.formField().

Reply:
Just tested it and yes you are right - in that case we should possibly remove it from _makeFileConfidential.scala.html_ (and perhaps the others where we have included it).

Reply:
I'm not sure if it does @tipperba looking at https://www.playframework.com/documentation/2.7.x/ScalaCsrf

Thanks @OC-Pomaroff that is a good spot

Reply:
I think this is the code called (don't shoot the messenger):

`play.filters.csrf.CSRF.getToken(request).map` {token => CSRF.formField(request)}`

Reply:
Turns out the answer is more complicated than this:

* Play has an object called `helper` which has a `form` function that doesn't include the CSRF token
* HMRC has an object called `uk.gov.hmrc.play.views.html.helpers` which has a `form` function which does include the token

Which one was used in a view template seems to have been decided by a coin flip so I've switched them all to the HMRC one.